src/Core/Framework/Gateway/Context/Command/AbstractContextGatewayCommand.php

```diff
@@ -0,0 +1,20 @@
+<?php declare(strict_types=1);
+
+namespace Shopware\Core\Framework\Gateway\Context\Command;
+
+use Shopware\Core\Framework\Log\Package;
+
+#[Package('framework')]
+abstract class AbstractContextGatewayCommand
+{
+    abstract public static function getDefaultKeyName(): string;
+
+    /**
+     * @param array<array-key, mixed> $payload
+     */
+    public static function createFromPayload(array $payload = []): static
+    {
+        /** @phpstan-ignore-next-line  */
```

Comment:
would you mind changing it to this? 
```suggestion
        /** @phpstan-ignore specific.identifier (comment why we need to ignore it)
```


also see https://phpstan.org/user-guide/ignoring-errors

So there is more context, why we have to ignore a PHPStan error and additionally only this specific error is ignored and not the whole line